lmp/jobserv.yml

```diff
@@ -88,6 +88,95 @@ triggers:
         persistent-volumes:
           bitbake: /var/cache/bitbake
 
+  - name: build-release-stable
+    type: git_poller
+    email:
+      users: 'ci-notifications@foundries.io'
+    params:
+      GIT_URL: |
+        https://github.com/foundriesio/lmp-manifest.git
+      GIT_POLL_REFS: |
+        refs/heads/zeus
+      OTA_LITE_TAG: postmerge-stable
+      AKLITE_TAG: promoted-stable
```

Comment:
ugh - forgot about the confusion on that. Should we call this tag "zeus" instead or is "stable" going to make the most sense?

Reply:
I was thinking stable because we will be based on different OE versions as we go, but we will only support one stable stream (which will be based on the latest OE stable).